diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7a0a6f1..7d33f3d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,8 +23,8 @@ if( NOT ${CMakeBuild_FOUND} )
   else()
     FetchContent_Declare(
       CMakeBuild
-      GIT_REPOSITORY https://github.com/NWChemEx-Project/CMakeBuild.git
-      GIT_TAG ${CMSB_TAG}
+      GIT_REPOSITORY https://github.com/edoapra/CMakeBuild.git
+      GIT_TAG ecpint
     )
   endif()
 
